package.json

```
@@ -1464,22 +1464,22 @@
                 },
                 {
                     "command": "aws.invokeLambda",
-                    "when": "view == aws.explorer && viewItem =~ /^(awsRegionFunctionNode|awsRegionFunctionNodeDownloadable|awsCloudFormationFunctionNode)$/",
+                    "when": "view == aws.explorer && viewItem =~ /^(awsRegionFunctionNode|awsRegionFunctionNodeDownloadable|awsCloudFormationFunctionNode|awsRegionFunctionNodeUploadable|awsRegionFunctionNodeDownloadableUploadable)$/",
                     "group": "0@1"
                 },
                 {
                     "command": "aws.downloadLambda",
-                    "when": "view == aws.explorer && viewItem =~ /^(awsRegionFunctionNode|awsRegionFunctionNodeDownloadable)$/",
+                    "when": "view == aws.explorer && viewItem =~ /^(awsRegionFunctionNode|awsRegionFunctionNodeDownloadable|awsRegionFunctionNodeUploadable|awsRegionFunctionNodeDownloadableUploadable)$/",
```

instead of adding all these items manually, can we use a regex like:


```suggestion
                    "when": "view == aws.explorer && viewItem =~ /^aws\\w*\\.(download|upload)$/",
```

Then to "turn on" download/upload functionality, when constructing the node we just append `.download` and/or `.upload`.

---

I think `/^aws\\w*\\.download/` is what we want right?